### DISCUSSION TITLE
About the page display corrrect or not with different start project.

### DISCUSSION THREAD

**Discussion by AnZeng00** [vikramlearning/blazorbootstrap#190] [General]
Hi,
I generate the blazor Webassembly app with visual studio 2022 template. Below is the solution structure:
![image](https://user-images.githubusercontent.com/52019144/233671248-3a90bd89-6388-4bf8-bf6c-fa3f91efa0da.png)
And I install Blazor.Bootstrap. After I finish the config, And I try to run the project. I found the page doesn't display correctly.
![image](https://user-images.githubusercontent.com/52019144/233672428-8a7fc159-2b27-42d3-b841-6918b7dc46df.png)
If I set the client project as start project, the page display correctly.
![image](https://user-images.githubusercontent.com/52019144/233673336-51a1559f-78b3-4122-ac82-e34edf667566.png)

Could you please help to tell me what is the problem?

**Comment by gvreddy04** [comment]
@AnZeng00 Did you enable the ASP.NET Core Hosted?

![image](https://user-images.githubusercontent.com/2337067/234502164-9ca1d7d4-978a-4ac5-b6e7-e9f17d756000.png)

Also provide below details:
.NET Version:
Blazor Bootsrap Version:

**Comment by AnZeng00** [comment]
Yes, I have enable this option.
